src/main/scala/shine/DPIA/showDPIA.scala

```diff
@@ -0,0 +1,86 @@
+package shine.DPIA
+
+import shine.DPIA.Phrases._
+import shine.DPIA.Types._
+import shine.DPIA.primitives.imperative._
+import shine.DPIA.primitives.functional._
+
+sealed trait ShowTypes
+case object On extends ShowTypes
+case object Off extends ShowTypes
+
+object showDPIA {
+  def apply[T <: PhraseType](p : Phrase[T]) : String = showDPIA().showPhrase(p)
+  def apply[T <: PhraseType](p : Phrase[T], showTypes : ShowTypes) : String = showDPIA(showTypes).showPhrase(p)
+}
+
+case class showDPIA(showTypes : ShowTypes = Off) {
+  val TAB : String = "    "
```

Comment:
maybe 2 spaces is enough and more concise width-wise?

Reply:
ok!